engine/src/cmdse.cpp

```diff
@@ -692,7 +684,19 @@ Parse_stat MCMessage::parse(MCScriptPoint &sp)
 			MCperror->add(PE_SEND_BADTARGET, sp);
 			return PS_ERROR;
 		}
-		return gettime(sp, &in, units);
+        
+        if (sp.skip_token(SP_FACTOR, TT_PREP, PT_AS) == PS_NORMAL)
+        {
+            if (!send ||
+                sp.skip_token(SP_FACTOR, TT_PROPERTY, P_SCRIPT) != PS_NORMAL)
+            {
+                MCperror->add(PE_SEND_BADAS, sp);
```

Comment:
@livecodeali is it bad that I want you to change the error to `PE_SEND_BADASS` :-P